.gitmodules

```diff
@@ -3,4 +3,4 @@
 	url = https://github.com/google/googletest
 [submodule "mavsdk-proto"]
 	path = proto
-	url = https://github.com/mavlink/MAVSDK-Proto.git
+	url = https://github.com/Daybreakerflint/MAVSDK-Proto
```

Comment:
Let me merge the MAVSDK-Proto PR, then you can update that and we can merge it.

Reply:
Done.